CONTRIBUTING.md

```diff
@@ -149,3 +140,50 @@ This repository also includes [stylecop ruleset
 files](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/master/build).
 These files are used to configure the _StyleCop.Analyzers_ which runs during
 build. Breaking the rules will result in a build failure.
+
+## Contributing a new project
+
+This repo is a great place to contribute a new instrumentation, exporter or
+any kind of extension. Although the projects within this repo share some
+properties and configurations, they are built and released independently.
+So if you are creating a new project within `/src` and corresponding test
+project within `/test`, here are a few things you should do to ensure that
+your project is automatically built and shipped through CI.
+
+* Include [`OpenTelemetry`](https://www.nuget.org/packages/OpenTelemetry)
```

Comment:
We cannot assume every contrib project must depend on the SDK. Instrumentations are only required to depend on Otel.API. (The DiagnosticSource instrumentations are an exception)

Reply:
Ah. I see what you mean. I notice that the writing a new instrumentation has been updated to use the ActivitySource API instead of DiagnosticSource. https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/extending-the-sdk/README.md#writing-own-instrumentation-library

Reply:
Yes. But most instrumentations need to depend on Otel.API, not just System.Diagnostics.DiagnosticSource alone.
Otel API provides:
Baggage 
Propagators
Builder extensions